src/Compilers/Core/MSBuildTask/AssemblyResolution.cs

```diff
@@ -54,10 +54,15 @@ private static bool TryRedirect(AssemblyName name)
                     return TryRedirect(name, s_b03f5f7f11d50a3a, 4, 0, 2, 0);
 
                 case "System.Diagnostics.StackTrace":
+                case "System.Security.AccessControl":
                     return TryRedirect(name, s_b03f5f7f11d50a3a, 4, 0, 3, 0);
 
             }
 
+#if DEBUG || BOOTSTRAP
+            ValidateBootstrap.AddFailedLoad(name);
```

Comment:
Actually, this won't work with our light-up code as it fails to look up desktop assemblies by design.